dj_redis_panel/redis_utils.py

```
@@ -1817,3 +1863,32 @@ def create_key(
+        Returns:
+            Dict with success status and any error information
+        """
+        try:
+            redis_conn = cls.get_redis_connection(instance_alias)
+
+            if flushall:
+                redis_conn.flushall()
+            else:
+                cls._select_db_if_not_cluster(redis_conn, instance_alias, db_number)
+                redis_conn.flushdb()
+            return {"success": True, "error": None, "message": f"Ok"}
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)